pom.xml

```diff
@@ -16,14 +16,19 @@
 
 	<properties>
 		<java.version>17</java.version>
+		<testcontainers.version>1.19.0</testcontainers.version>
 	</properties>
 
     <dependencyManagement>
         <dependencies>
+			<!--
+				You don't need to configure testcontainers-bom
+				if you are using Spring Boot 3.1.0+
```

Comment:
only if the Testcontainers latest version doesn't match with the one provided by SB